src/CSSList/CSSList.php

```diff
@@ -263,7 +263,7 @@ public function getLineNo()
      *
      * @return void
```

Comment:
We can also drop all those `@return void` PHPDoc annotations now.

Reply:
PHPDoc annotations are gone.